docs/content/any/guides/new-roles/_index.md

```diff
@@ -41,7 +41,7 @@ The roles library includes:
 
 ## Get started
 
-The Oso Roles library is coming soon for {{< lang >}}! {{< /lang >}}
+The Oso Roles library is coming soon for {{< lang >}}!
```

Comment:
Off-topic: Is "Oso Roles" what we've settled on calling the feature? I would call it a "feature" instead of a "library" — latter sounds like something you'd need to install separately instead of a separate module within the same package.

Reply:
Yeah, I prefer feature. We should cover this if we can in #980 .